coast/stats_util.py

```diff
@@ -168,7 +167,7 @@ def doodson_x0_filter(elevation, ax=0):
         Filtered array of same rank as elevation.
     """
     if elevation.shape[ax] < 39:
-        print("Doodson_XO: Ensure time axis has >=39 elements. Returning.")
+        print("Doodson_XO: Ensure time axis has >=39 elements. Returning.")  # TODO Does this really need to be printed?
```

Comment:
It should probably be an error message?

Reply:
Changed this to be an error log instead now.